app/models/katello/package_group.rb

```diff
@@ -1,7 +1,30 @@
 module Katello
-  class PackageGroup
+  class PackageGroup < Katello::Model
+    include Glue::Pulp::PulpContentUnit
     include Glue::Pulp::PackageGroup if Katello.config.use_pulp
-    include Glue::ElasticSearch::PackageGroup if Katello.config.use_elasticsearch
+
     CONTENT_TYPE = "package_group"
+
+    has_many :repositories, :through => :repository_package_groups, :class_name => "Katello::Repository"
```

Comment:
chatted with the pulp team about this and they don't see it as an issue.  And i would agree there isn't a ton of benefit to introduce entirely new structures to support this association type.